ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/cloud_trace_propagator.py

```diff
@@ -28,7 +28,7 @@
 )
 
 _TRACE_CONTEXT_HEADER_NAME = "X-Cloud-Trace-Context"
-_TRACE_CONTEXT_HEADER_FORMAT = r"^(?P<trace_id>[0-9a-f]{32})\/(?P<span_id>[\d]{20});o=(?P<trace_flags>\d+)$"
+_TRACE_CONTEXT_HEADER_FORMAT = r"^(?P<trace_id>[0-9a-f]{32})\/(?P<span_id>[\d+]{1,20});o=(?P<trace_flags>\d+)$"
```

Comment:
Technically, this now accepts any length integer; either use `\d+` or `\d{1,20}`

Reply:
This should accept only integers of lengths between 1 and 20

Reply:
That's incorrect, actually, fixed now.